### ISSUE TITLE
Similar Model - During Training, all predictions are close to mean of training data

### ISSUE THREAD

**Issue by ghost** [zhouhaoyi/Informer2020#354]
Hello Authors

I am trying to implement a similar model but with a graph attention network in front of the transformer to capture spatial correlations. However, during training, I am not able to capture the trend in the data for each of the network elements. (PEMS-BAY) dataset.

Could you please help with a one-liner comment on the two questions below?

1 - When training, how do the predictions from the final layer of the model look? Do they capture the trend of the input data or produce values close to the mean value of the training data?
2 - During training, what does the training loss look like in your experiments - is it lower than the final MAE reported for 60 min interval? 

during training, the final layer outputs (n_timesteps, n_sensors, 1) value in one go while in testing, it does 1, n_sensors,1 for the horizon number of timesteps. In general, test loss and training loss need to converge with epochs of training, but test loss is generally slightly higher than the training loss - does that apply to this auto-regressive model? (not applicable to Informer as it does a one-shot prediction, but in general for a transformer)

**Comment by zhouhaoyi** [comment]
We recommend our similar work at https://github.com/zhouhaoyi/POLLA
